bot/cogs/tags.py

```diff
@@ -205,12 +207,22 @@ def _command_on_cooldown(tag_name: str) -> bool:
                         "time": time.time(),
                         "channel": ctx.channel.id
                     }
-                await ctx.send(embed=Embed.from_dict(tag['embed']))
+                await wait_for_deletion(
```

Comment:
Do we really want to delete successful tag hits?

Reply:
I think this may help in a situation when a user accidentally uses a command in wrong channel. And this is just a small reaction under a message, nothing big and breaking so I think this should okay.